src/pages/workspaces/workspace/Data.js

```diff
@@ -1379,6 +1380,25 @@ export const WorkspaceData = _.flow(
                   undefined,
                   () =>
                     Utils.cond(
+                      [
+                        isAzureWorkspace && wdsError,
+                        () =>
+                          div(
+                            {
+                              style: { textAlign: 'center', lineHeight: '1.4rem', marginTop: '1rem', marginLeft: '5rem', marginRight: '5rem' },
+                            },
+                            [
+                              'Oh no! Something went wrong while preparing your data tables.',
+                              div({}, [
```

Comment:
Nit: empty props can be omitted.

```suggestion
                              div([
```

Reply:
Removed.